### DISCUSSION TITLE
Which type of printer did you use?

### DISCUSSION THREAD

**Discussion by los19** [LorenceChen/DHO800_BatteryPack#1] [Q&A]
Hey LorenceChen, awesome Project!
I'm struggeling to print the Case. What type of printer did you use to get such a clean finish? I cant find a orientation to place the model on the buildplate on my fdm printer because every placement has massive overhangs.

Thanks for your work!

**Comment by LorenceChen** [comment]
Printed by a third party, the printer should be HP MJF, Nylon, The accuracy is ±0.3mm. And the surface sandblasting.
发自我的 iPhone

在 2024年3月17日，上午12:23，los19 ***@***.***> 写道：

﻿

Hey LorenceChen, awesome Project!
I'm struggeling to print the Case. What type of printer did you use to get such a clean finish? I cant find a orientation to place the model on the buildplate on my fdm printer because every placement has massive overhangs.

Thanks for your work!
